commands/utility/brawlstars/playerinfo.py

```
@@ -1,46 +1,13 @@
-from typing import Any
+from __future__ import annotations
 
-import aiohttp
-from aiohttp import ClientTimeout
+from typing import Any
```

Flake8 (7.3.0)

[error] 'typing.Any' imported but unused

(F401)